wise_pizza/explain.py

```diff
@@ -224,6 +229,7 @@ def explain_changes_in_totals(
             force_dim="Change from" if how == "force_dim" else None,
             force_add_up=force_add_up,
             constrain_signs=constrain_signs,
+            cluster_values=cluster_values,
```

Comment:
How about adding cluster_values parameter also to the explain_changes_in_averages? Of course we everytime call explain_changes_in_totals then, but just to be sure if someone wants to use cluster_values = False parameter

Reply:
Nice catch, will fix